### PR TITLE
NXP-29200: add user/group events in audit directories

### PR DESCRIPTION
**Pull request by michaelva**

### CODE REVIEW

**Comment by nmpcunha** [comment]
I believe this is also related to https://jira.nuxeo.com/browse/NXP-29175 (But not sure when it would be tackled)

**Comment by michaelva** [comment]
@nmpcunha I believed it is slightly different here. This PR is to make the category and event types available in the UI filters which are directory suggestion widgets

File: modules/platform/nuxeo-platform-audit/nuxeo-platform-audit-core/src/main/resources/directories/event-types.csv [hunk 1]
```diff
@@ -37,4 +37,10 @@ id,label,obsolete,ordering
...
 "lifecycle_transition_event","lifecycle_transition_event","0","390"
 "annotationCreated","annotationCreated","0","400"
-"documentWaitingPublication","documentWaitingPublication","0","410"
+"documentWaitingPublication","documentWaitingPublication","0","410",
+"user_created","user_created","0","420",
+"user_modified","user_modified","0","430",
+"user_deleted","user_deleted","0","440",
+"group_created","group_created","0","450",
+"group_modified","group_modified","0","460",
+"group_deleted","group_deleted","0","470"
```
Comment (troger) [inline modules/platform/nuxeo-platform-audit/nuxeo-platform-audit-core/src/main/resources/directories/event-types.csv:46]: Missing newline.

**Comment by michaelva** [comment]
@NourNuxeo I always squash when merging

**Comment by NourNuxeo** [comment]
not sure it's useful to keep "NXP-29200: add new line eof" in the final commit message 🤷‍♂️

**Comment by kevinleturc** [comment]
T&P needed.

**Comment by NourNuxeo** [comment]
ran it 3 times without success:
https://qa.nuxeo.org/jenkins/view/TestAndPush/job/TestAndPush/job/ondemand-testandpush-nalkotob/159
https://qa.nuxeo.org/jenkins/view/TestAndPush/job/TestAndPush/job/ondemand-testandpush-nalkotob/160
https://qa.nuxeo.org/jenkins/view/TestAndPush/job/TestAndPush/job/ondemand-testandpush-nalkotob/161

The tests are ok but some errors are logged consistently. You need to check in the log files. The console points this in particular:
```08:29:03 /opt/jenkins/workspace/TestAndPush/ondemand-testandpush-nalkotob/ftests/nuxeo-3d-ftests/target/tomcat/log/server.log:
08:29:03 2020-07-02T03:51:16,604 ERROR [main] [org.nuxeo.ecm.directory.DirectoryCSVLoader] Invalid column count while reading CSV file: nxblob-9564763030034943956.tmp, line: 7```

**Comment by michaelva** [comment]
thanks @NourNuxeo I have tried several times too and same result on my side.

**Comment by ataillefer** [comment]
@NourNuxeo we've just solved it, error logs were indeed relevant, problem was some extra commas at the end of the CSV lines.

**Comment by michaelva** [comment]
thanks a lot @ataillefer for noticing. I pushed an update and started a new T&P

**Comment by michaelva** [comment]
test and push passed: https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-mvachette/16/

**Comment by troger** [comment]
@michaelva anything blocking you to merge the PR?

**Comment by troger** [comment]
I've rebased the branch on `origin/master` to get up to date statuses.

**Review by ataillefer — APPROVED** [review]

**Review by kevinleturc — APPROVED** [review]

**Pull request merged**